### PR TITLE
Experimental translation

### PR DESCRIPTION
**Pull request by agektmr**
This pull request is to see if l10n will work on our new infrastructure.

### CODE REVIEW

**Comment by robdodson** [comment]
Just started reviewing :)

**Comment by robdodson** [comment]
It looks like we'll need to do some refactoring of the build scripts to make this work. I saw that they generated a `build/en` directory but I didn't notice that they don't have a `content/en` directory. Instead the build script always looks in the root of the `content/` directory and only generates `build/en`

https://github.com/GoogleChrome/web.dev/blob/39006c92c4067a7b4562a539f806149dcea6d7dd/lib/index.js#L39-L40

I think I can still go ahead and create a CL on DevSite so we can test, I'll just manually create the `build/ja` folder and put the content in there.

@TimvdLippe I think this is something we'll want to consider for the refactoring you're doing. We should probably have a directory structure like this:

```
content/
  en/
  ja/
  es/
  ...
```

with translations living in those subdirectories.

Tim—We should probably also move `content/images` since every language will want to link to those images but it doesn't make sense to duplicate them. Maybe that images directory should live in `contributors/` ?

File: content/ja/accessible/control-focus-with-tabindex/index.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+---
+page_type: guide
+title: tabinxed でフォーカスを制御する
```
[the remark this answers is not in the document]
Comment (agektmr) [reply content/ja/accessible/control-focus-with-tabindex/index.md]: Fixed

File: content/ja/accessible/control-focus-with-tabindex/index.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  <button> や <input> といったネイティブの HTML 要素には、キーボードアクセシビリティ機能が最初から組み込まれています。 カスタムのコンポーネントを開発している場合は、tabindex 属性でキーボードでアクセシブルになるようにしましょう。
+author: robdodson
+web_lighthouse:
+  - tabindex
+  - focusable-controls
+web_updated_on: 2018-12-06
+web_published_on: 2018-11-18
+wf_blink_components: Blink>Accessibility
+---
+# tabinxed でフォーカスを制御する
```
[the remark this answers is not in the document]
Comment (agektmr) [reply content/ja/accessible/control-focus-with-tabindex/index.md]: Fixed.

**Comment by agektmr** [comment]
@robdodson Let me know if you want me to move contents to `/en` directory.

**Comment by TimvdLippe** [comment]
Yeah I can manage that. Will include it in the prototype :+1:

**Comment by robdodson** [comment]
I just tried this out on my test server and it looks like the content is showing up if my language is set to japanese, but the page template is rendering a little broken (see the grey bar on the right). I'll have to do some work to figure out why that is.

image

**Comment by agektmr** [comment]
@robdodson do you plan to change directory structure from `/content/<actual content>` to `/content/<lang>/<actual content>` anytime soon?

**Comment by robdodson** [comment]
@agektmr I think we should do this in the branch that @devnook is working on.
https://github.com/GoogleChrome/web.dev/pull/616

**Comment by robdodson** [comment]
I'm going to close this because it's based on an older version of our site. We're definitely still planning to work on translation but I'm just doing a bit of house cleaning for the upcoming quarter.

**Pull request closed without merging**